stable/insights-agent/values.yaml

```diff
@@ -121,6 +121,7 @@ opa:
   image:
     repository: quay.io/fairwinds/fw-opa
     tag: "0.2"
+  additionalAccess:
```

Comment:
Is this valid YAML? I thought it needed to be commented if left blank

Reply:
Yeah this is valid YAML. It's one of the approved ways to do a null. It's weird looking though :)